subprojects/docs/src/docs/userguide/authoringMaintainableBuildScripts.adoc

```diff
@@ -157,6 +157,28 @@ Now that the build logic has been translated into a plugin, you can apply it in
 </sample>
 ++++
 
+[[sec:avoiding_use_of_gradlebuild]]
+=== Avoiding the use of `GradleBuild`
+
+A build script can define task for invocation other Gradle builds through the task type api:org.gradle.api.tasks.GradleBuild[]. The use of this type is generally discouraged. There are some corner cases where the invoked build doesn't expose the same runtime behavior as from the command line or through the Tooling API leading to unexpected results.
```

Comment:
single line per sentence?

Reply:
first sentence is a bit awkward.  maybe:  "The api:org.gradle.api.tasks.GradleBuild[] task type allows a build script to define a task that invokes another Gradle build."

Reply:
done and done